documentation/content/en/books/fdp-primer/writing-style/_index.adoc

```diff
@@ -187,6 +187,32 @@ All human beings are born free and equal in dignity and rights.
 They are endowed with reason and conscience and should act towards one another in a spirit of brotherhood.
 ....
 
+[[sentence-spacing]]
+=== Spaces between sentences
+
+There are two conventions.
+
+*Generally*: 
+
+* one space. 
+
+This generalization includes monospaced representation of monospaced texts that do _not_ follow the two spaces convention.
+
+Where a variable-width font will be used: the per-line convention, above, will usually negate the need to think about spacing. 
+
+If a multi-sentence list item does use a space between sentences (with the possibility of strictly replacing the space with a line break): 
+
+* consider edition, to have no more than one sentence per point
+** do not assume that nesting of points is the solution
+** instead, an adjacent bullet point might be promoted to a simple paragraph.
+
+*Exceptionally*: 
+
+* two spaces in errata notices
+* two spaces in security advisories
+* two spaces in manual pages (in the `src` tree, for example)
+* two spaces in other files that will normally be viewed with a monospaced font.
```

Comment:
### Spacing between sentences

Recent variations: 

- https://github.com/freebsd/freebsd-src/commit/21850106fdda5269bc881f0e62839dff3d9edf47#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R31-R35
- https://github.com/freebsd/freebsd-src/commit/3a1f834b5228986a7c14fd60da13cf2700e80996#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R31-R34
- https://github.com/freebsd/freebsd-src/commit/564c5314f6d9aa2b20ebd437b86fe441e967ada1#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R31-R33
- https://github.com/freebsd/freebsd-src/commit/319d2bf407b3762da6f1c67ffe8dce2fee587aaf#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R31-R34
- https://github.com/freebsd/freebsd-src/commit/f475b7108bfec35125a4bc611c64faa85c8b79d0#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R31-R35
- https://github.com/freebsd/freebsd-src/commit/4722ceb7d53e76507c76e053caab6b6f7b24ecef#diff-7a99811ebf6b6268a2581fcf050df0a2c325fdbf1c02481dad16afd38f4bd6a1R36-R40

Reply:
@bsdimp @dag-erling @emaste 

For `UPDATING`: I assume that wherever there is a pair of sentences, **two spaces between the pair** should be the convention. 

Context: 

> … files that will normally be viewed with a **monospaced** font.

- thanks to Mike Karels for dropping a hint. 

If you believe otherwise, please let me know. 

Thanks

Reply:
> **two spaces between the pair** should be the convention.

Resolved with <https://lists.freebsd.org/archives/dev-commits-src-main/2023-August/017894.html>: 

> UPDATING strongly prefers two spaces …

– thanks